crates/binary-analysis/src/rtti.rs

```diff
@@ -0,0 +1,263 @@
+use std::{collections::HashMap, ffi::CStr, ops::Range, ptr};
+
+use pelite::pe::{
+    msvc::{
+        RTTIBaseClassDescriptor, RTTIClassHierarchyDescriptor, RTTICompleteObjectLocator,
+        TypeDescriptor, PMD,
+    },
+    Pe, Rva, Va,
+};
+use rayon::{
+    iter::{IntoParallelIterator, ParallelIterator},
+    slice::ParallelSlice,
+};
+use thiserror::Error;
+
+use crate::pe::sections;
+
+#[derive(Error, Debug)]
+pub enum RttiError {
+    #[error(transparent)]
+    Pelite(#[from] pelite::Error),
+    #[error("PE section \"{0}\" is missing")]
+    Section(&'static str),
+    #[error("index out of bounds")]
+    Bounds,
+}
+
+pub type ClassMap = HashMap<Box<str>, Box<[UntypedVmt]>>;
+
+#[derive(Clone, Copy, Debug)]
+pub struct UntypedVmt(*const Va);
+
+#[derive(Clone)]
+pub struct ClassCol<'a, P>
```

Comment:
Col?

Reply:
Can we name this something else? It's not an immediately obvious acronym. `ClassRttiData` or something would be fine